oh_staff_ui/management/commands/create_mods_records.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from django.core.management.base import BaseCommand
+from oh_staff_ui.classes.OralHistoryMods import OralHistoryMods
+from oh_staff_ui.models import ProjectItem
+
+# For handling command-line processing
+from django.contrib.auth.models import User
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_mods_records(item_id: int) -> None:
+    """Given an item_id write the related MODS record to public location"""
+    pi = ProjectItem.objects.filter(id=item_id).first()
```

Comment:
`id` is always unique.  The `filter().first()` idiom avoids exceptions, but the user has no info whether this operation succeeded or failed.

It's better to use `get()`, and catch the `ProjectItem.DoesNotExist` exception with a useful message to the user.

Reply:
Swapped `filter().first()` for `get()`, added exception handling and user error message for `ProjectItem.DoesNotExist` case